ITP-JAN-25 | Sulaiman Kamara | Module-Onboarding | Sprint 2

sul-devs opened the pull request:
I have implemented the form with built-in HTML validation.

Waldo-Strydom commented:
Please add your cohort name e.g South Africa to your PR titles.

Please add labels to your PRs

sul-devs commented:
> Please add your cohort name e.g South Africa to your PR titles.
> 
> Please add labels to your PRs

Hi would that be my location, Like London, because I've added cohort ITP-JAN-25 in the title? How do I add a label here, I added in my coursework planner, but not sure how to add one here?

Waldo-Strydom commented on Form-Controls/index.html:
```diff
@@ -14,14 +14,88 @@ <h1>Product Pick</h1>
+          />
+        </div>
+
+        <!-- Last Name field -->
```
last name is not needed.

Waldo-Strydom commented on Form-Controls/index.html:
```diff
@@ -14,14 +14,88 @@ <h1>Product Pick</h1>
+            type="text"
+            id="first-name"
+            name="first-name"
+            placeholder="First Name"
```
You need to require a min of 2 characters

Waldo-Strydom commented:
> > Please add your cohort name e.g South Africa to your PR titles.
> > Please add labels to your PRs
> 
> Hi would that be my location, Like London, because I've added cohort ITP-JAN-25 in the title? How do I add a label here, I added in my coursework planner, but not sure how to add one here?

It's important to add your location because London volunteers mark the work of London trainees.

sul-devs commented:
Hi I've made the suggested changes. Can you please review my 2 latest commits?

Waldo-Strydom commented:
Well Done.
